src/Tools/dotnet-monitor/LoggingEventIds.cs

```diff
@@ -81,7 +81,10 @@ internal enum LoggingEventIds
         DiagnosticPortWatchingFailed = 68,
         InvalidMetadata = 69,
         DuplicateKeyInMetadata = 70,
-        EnvironmentVariableNotFound = 71
+        EnvironmentVariableNotFound = 71,
+        FailedInitializeSharedLibraryStorage = 73,
+        UnableToApplyProfiler = 74,
+        SharedLibraryPath
```

Comment:
Is there a reason this doesn't have a value? I'm also assuming skipping 72 was intentional to avoid the merge conflict with `EnvironmentBlockNotSupported `

Reply:
Fixed by adding explicit value; the skipping was intentional, but there will probably be a merge conflict anyway